packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -184,73 +207,64 @@ const DialpadContainer = (props: {
   onSendDtmfTone?: (dtmfTone: DtmfTone) => Promise<void>;
   /**  Callback for dialpad button behavior */
   onClickDialpadButton?: (buttonValue: string, buttonIndex: number) => void;
-  /**  customize dialpad input formatting */
-  onDisplayDialpadInput?: (input: string) => string;
+  /** Pass in custom content to dialpad textfield */
```

Comment:
should this comment maybe be something like :
`/** Access to the text field within dialpad */`

Reply:
value passed in here will directly set the content in the textfield, so it should not be called 'access'?

Reply:
oh sorry I was thinking about accessing that value, they can still do that through onChange right?